resources/rules/gcp-iam-and-security-ensure-only-gcp-managed-service-account-keys-for-each-service-account.yaml

```diff
@@ -1,4 +1,4 @@
-id: 419ab56b-0837-4bc8-a04d-b0e184dc2303
```

Comment:
I can't find any rules in PG with this UUID. This makes me think it's safe to randomly changed one of the duplicate IDs but we should verify first.